src/lib/webpack/webpack-base-config.js

```diff
@@ -220,7 +220,7 @@ export default (env) => {
 		// produce HTML & CSS:
 		addPlugins([
 			new ExtractTextPlugin({
-				filename: 'style.css',
+				filename: isProd ? "style.[contenthash:5].css" : "style.css",
```

Comment:
The other option is to expose a `mainfest.json` which would be used to resolving filenames from entries (dunno how this works - only seen folks doing that)